data/editor.config

```diff
@@ -931,6 +961,9 @@
       <type id="tourism-viewpoint">
         <include field="name" />
       </type>
+      <type id="tourism-zoo">
```

Comment:
Aren't most zoos already mapped? They are usually located in larger cities with OSM coverage.